### ISSUE TITLE
Improve `get_userlist()` function by reducing data saved in the array and only store it when needed

### ISSUE THREAD

**Issue by Wolfsblvt** [Wolfsblvt/mentions#1]
From this comment: https://github.com/Wolfsblvt/mentions/commit/02d5158a5da1c9e7ad8cf994f9249b330cafa1d1#commitcomment-9579668

Improve `get_userlist()` function by reducing data saved in the array and only store it when needed

**Comment by Wolfsblvt** [comment]
@Elsensee Let's see how far I can get here :P

**Comment by Wolfsblvt** [comment]
Important points for fixing this mentioned here: https://www.phpbb.com/community/viewtopic.php?p=13937476#p13937476

**Comment by gn36** [comment]
I would think that #10 would pretty much solve this also.

**Comment by Wolfsblvt** [comment]
Yeah, that would improve that too. But it still must be parsed once, on post submit. So this should be improved too I would say.

But you are right, most of the issues are connected in some way :D

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This project is no longer maintained, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.

Anyone interested in continuing development is welcome to do so through a fork, subject to the repository’s license and any applicable third-party licenses.
